core/src/main/java/job/scheduler/core/JobScheduler.java

```diff
@@ -0,0 +1,30 @@
+package job.scheduler.core;
+
+import job.scheduler.core.config.ServerConfig;
+import job.scheduler.core.server.JobSchedulerServer;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Entry class for Job scheduler.
+ */
+@Slf4j
+public class JobScheduler {
+
+  private static ServerConfig getConfigFromProps() {
+    // TODO: read from props
+    return new ServerConfig(String.valueOf(1), "localhost:2181", 18000);
```

Comment:
Add yml config file and read it here. Instead of hardcoding.

Reply:
+1